src/hazelcore/parse/dune

```diff
@@ -0,0 +1,14 @@
+(include_subdirs no)
```

Comment:
Since it can be its own library, lets move the parser out to the `src` directory instead of it being inside `hazelcore`. You can call it `hazeltext`.